packages/verifiable-credentials/src/credentials.ts

```diff
@@ -204,15 +216,24 @@ export default class Credentials {
 		}
 
 		if (options && options.issuanceDate) {
+			// validate ISO UTC date format 
+			const regex = /\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}.\d{3}Z/
```

Comment:
Instead of doing this. Can you use the `dayjs` library to convert the string to a UTC time and then ensure it always generates the correct string format?

Reply:
This also needs a unit test.

Reply:
modified this logic and added a test case

Reply:
implemented